l5kit/l5kit/sampling/agent_sampling.py

```diff
@@ -47,48 +135,44 @@ def generate_agent_sample(
         selected_track_id (Optional[int]): Either None for AV, or the ID of an agent that you want to
         predict the future of. This agent is centered in the raster and the returned targets are derived from
         their future states.
-        raster_size (Tuple[int, int]): Desired output raster dimensions
-        pixel_size (np.ndarray): Size of one pixel in the real world
-        ego_center (np.ndarray): Where in the raster to draw the ego, [0.5,0.5] would be the center
+        render_context (RenderContext): The context for rasterisation
```

Comment:
I actually wonder if render context should be removed, or Optional, since rasterizer can be optional. In any case seems like this is not in scope for this PR.

Reply:
Probably yes (it's also inside the rasteriser anyway)